app/src/main/java/org/eyeseetea/malariacare/domain/entity/SurveyConflict.java

```diff
@@ -0,0 +1,28 @@
+package org.eyeseetea.malariacare.domain.entity;
+
+public class SurveyConflict {
```

Comment:
@idelcano really uid from this class not is survey id then I think a name as PushConflict or simply Conflict is more appropriate